src/main/java/org/reaktivity/nukleus/http2/internal/types/stream/FrameType.java

```diff
@@ -28,19 +28,19 @@
     WINDOW_UPDATE(8),
     CONTINUATION(9);
 
-    private final byte type;
+    private final int type;
 
     FrameType(int type)
     {
-        this.type = (byte) type;
+        this.type = type;
     }
 
-    public byte getType()
+    public byte type()
```

Comment:
Do we need `type()` as an `int` or ` byte`?
Doesn't seem quite right to be casting on every access when we control the storage type.